src/middlewared/middlewared/plugins/init_shutdown_script.py

```diff
@@ -22,6 +24,7 @@ class Config:
         Str('when', enum=['PREINIT', 'POSTINIT', 'SHUTDOWN'], required=True),
         Bool('enabled', default=True),
         Int('timeout', default=10),
+        Str('comment', default=''),
```

Comment:
Range validator instead of that and COMMENT_LENGTH?

Reply:
Ah, that doesnt work for Str, I guess?

Reply:
Good point, it doesn't but we should adapt it to work for string values as well checking their length if the value is str else integer it already is